### PR TITLE
allow different lang settings for multi queries

### PR DESCRIPTION
**Pull request by Razzmatazzz**
Currently, if a user executes multiple queries in a single API request, the lang setting for the first query overrides any lang settings for the following queries. This fixes that so each query can have its own lang argument.

### CODE REVIEW

**Review by GrantBirki — APPROVED** [review]

File: utils/graphql-util.js [hunk 1]
```diff
@@ -15,7 +15,14 @@ module.exports = {
     getLang: (info) => {
         let lang = 'en';
         let langFound = false;
+        let myRoot = info.path.key;
+        for (let currentNode = info.path.prev; currentNode; currentNode = currentNode.prev) {
+            myRoot = currentNode.key;
+        }
         for (const selection of info.operation.selectionSet.selections) {
+            let selectionRoot = selection.name.value;
+            if (selection.alias) selectionRoot = selection.alias.value;
```
Comment (Shebuka) [inline utils/graphql-util.js:24]:
```suggestion
            if (selection.alias)
                selectionRoot = selection.alias.value;
```

File: utils/graphql-util.js [hunk 2]
```diff
@@ -15,7 +15,14 @@ module.exports = {
...
+            if (selectionRoot !== myRoot) continue;
```
Comment (Shebuka) [inline utils/graphql-util.js:25]:
```suggestion
            if (selectionRoot !== myRoot)
                continue;
```

**Review by Shebuka — COMMENTED** [review]
Please do not put `if` and result of the `if` on the same row, really hard to read

**Comment by Razzmatazzz** [comment]
.lock --reason until approved and merged, we don't want other branches deployed because it will re-introduce a bug that breaks parts of the website when viewed in non-english languages

**Comment by Razzmatazzz** [comment]
.lock

**Review by GrantBirki — APPROVED** [review]

**Comment by GrantBirki** [comment]
Wowzers you have discovered an annoying bug in the `branch-deploy` Action...

```text
.lock --reason until approved and merged, we don't want other branches deployed because it will re-introduce a bug that breaks parts of the website when viewed in non-english languages
```

_enhance_

```text
it will re-introduce
```

_enhance!_

```text
re-introduce
```

_enhance!!_

```text
-i
```

Bingo..

```js
// Lock info flags
const LOCK_INFO_FLAGS = ['--info', '--i', '-i', '-d', '--details', '--d']
```

Since `-i` counts as a `LOCK_INFO_FLAGS`, the Action is only returning lock info because your message contains `-i` in `re-introduce`. I'll have to fix this bug 🐛 

[source code](https://github.com/github/branch-deploy/blob/1fa2801ec04617e39a9f379ef709b01d89c7d145/src/main.js#L24)

**Comment by GrantBirki** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
lol just doing my part

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
.unlock

**Pull request merged**